addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -8,30 +8,46 @@ const utils = require('web.utils');
 
 const PopupWidget = publicWidget.Widget.extend({
     selector: '.s_popup',
-    events: {
+    read_events: {
         'click .js_close_popup': '_onCloseClick',
         'hide.bs.modal': '_onHideModal',
         'show.bs.modal': '_onShowModal',
     },
+    disabledInEditableMode: false,
```

Comment:
Sometimes I judge it ok but for the popup widget, I would really not do that in stable. You can imagine 100 things custom code do to this widget in stable. This would enable those in edit mode too, potentially breaking everything. I just merged a fix of @rdeodoo which introduced a second widget definition for popups with `disabledInEditableMode: false` here: https://github.com/odoo/odoo/commit/cfd53b8fae3ad9f677df49a03fe6d2945dbb5da2#diff-b41deeb2a91df218ee83ece1493beb5091bbfea9168822c64147ebbc6e480298R10 => Rebase your branch and extend that one... EDIT: actually your branch here is for 14.0 but RDE's fix was for 15.0. No worries: just create the same class RDE did at the same position (but only do what is necessary for you of course).

(We'll probably regroup both in master).